govuk-one-login/authentication-api
# BAU: Remove doc app handling for auth StartHandler

## What

Remove doc app handling for auth StartHandler

No doc app traffic is received by auth anymore, it is all handled by orch. By default return docCheckingAppUser=false to the frontend pending removal of the frontend doc app code.

## How to review


1. Code Review
1. Deploy to a dev/sandpit environment.
1. Check that sign in and sign up scenarios work correctly.

## Checklist
